data/scenarios/Tutorials/bind.yaml

```
@@ -6,9 +6,9 @@ objectives:
       - |
         Every command returns a value.  However, some simple commands, like
         `move`, do not have any meaningful
-        value to return.  Since the unit type `()` has only one value, also
+        value to return.  Since the unit type `unit` has only one value,
```

Should we mention the motivation for the syntax here?
```suggestion
        - You can think of `()` as an empty tuple, so that a pair `(false,2)` has two meaningful components, normal value `(true)` has one component and the unit `()` has none.
```

---

Hmm, at this point we haven't talked about tuples in Swarm at all, although most people are probably familiar with the idea of tuples in general...

---

I'm thinking at this point explaining the motivation behind the syntax would just be extra noise for a lot of people.   They are probably happy just being told that it exists.  I think later if we make a tutorial going more in depth into algebraic data types (#664) that would be the place to explain the notation for `()`.

However, your suggestion made me take a close look at this tutorial challenge and I think I have improved it in several other ways.

---

Nice, I like it :+1: